tests/functional/pv/pv_services/test_pvc_creation_in_megabytes.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+import logging
+from ocs_ci.framework.testlib import bugzilla, tier1
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.pytest_customization.marks import green_squad
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@bugzilla("2239208")
+@green_squad
+@pytest.mark.polarion_id("OCS-5476")
+class TestPvcCreationInMegabytes:
+    """
+    1. This class takes care of creating Cephfs PVC with small size 499M, 500M, 501M.
+    2.Passing size_unit as 'M' [mega-bytes] and the created PVC size will be displayed in 'Mi' format in pvc yaml.
+    3.Verifying the created PVC size by converting it from 'Mi' [mebi-bytes] to 'bytes'
+    and comparing it with the given size.
+    4. The expectation here is the created PVC sized should be greater than or equal to the given size.
+    It should not be lesser than the given size.
+    """
+
+    def test_pvc_creation_in_megabytes(self, pvc_factory):
+        """
+        This function facilitates
+        1. Create PVC in Cephfs with size_unit as "M" . Used access mode is RWX.
+        2. Convert the PVC's size from 'Mi' to bytes.
+        3. Validate created PVC size with the given_pvc_size in "M" format after converting it into bytes.
+        """
+        access_mode = constants.ACCESS_MODE_RWX
+        interface = constants.CEPHFILESYSTEM
```

Comment:
same as my above comment - you dont need this var, you can directly pass it to the func